src/yunohost/data_migrations/0009_dh_params.py

```diff
@@ -0,0 +1,28 @@
+import re
+import os
+
+from moulinette.utils.log import getActionLogger
+from moulinette.utils.filesystem import chown
+
+from yunohost.tools import Migration
+from yunohost.service import service_regen_conf
+
+command = "nice -n 19 openssl dhparam -out /etc/ssl/private/dh2048.pem -outform PEM -2 2048 -dsaparam 2> /var/log/yunohost/dhparam_generation.log && chown root:ssl-cert /etc/ssl/private/dh2048.pem && yunohost service regen-conf >> /var/log/yunohost/dhparam_generation.log && rm /etc/cron.hourly/yunohost-generate-dh-params\n"
```

Comment:
Do we really want to run that in hourly and not in daily instead?

Reply:
You also probably want to "%s" in the path the file to remove once it has been declared in this file for stronger code.

Reply:
don't understand you lst comment. 

For hourly, for me it's quite correct. If we set it to daily, the user will have to wait 1 day whereas it's quite long. Moreover we said that hourly was ok.

Reply:
I didn't understand : "You also probably want to "%s" in the path the file to remove once it has been declared in this file for stronger code.". Could you explain me what is the goal of adding %s ?

And, I wanted to know if I have to set it to daily or hourly as we had a conversation about that and chose hourly. So, I really don't know what to do.